cmd/edgectl/aes_install.go

```diff
@@ -80,6 +80,15 @@ func aesInstall(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	// Grab Ambassador's install ID as the cluster ID we'll send going forward
+	for {
+		if clusterID, err := i.CaptureKubectl("get cluster ID", "-n", "ambassador", "exec", "deploy/ambassador", "python3", "kubewatch.py"); err == nil {
```

Comment:
Would it be safer to reproduce the logic from `kubewatch.py` rather than trying to exec into a running pod? Reproducing the logic simply means reading metadata information about the namespace.

Also, should the namespace `-n ambassador` be a variable?

Reply:
No, because part of the point here is that the Ambassador pod is running (though not necessarily Ready), which I'm considering good enough to for the "deploy" metrics report. I will point that out in a comment.

Also, no. The installer always installs into the ambassador namespace, always creates one pod, always has AES managing the whole cluster, etc. This is not intended to be a configurable installation, rather an opinionated one. The story document doesn't say this clearly, but that from talking to Rafi, that was his intent.

Reply:
Note from Slack discussion:

> The installer always installs into the ambassador namespace, always creates one pod, always has AES managing the whole cluster, etc. 

This assumption also true for detecting existing Helm installs, in the sense that an existing install in a different namespace will be ignored.